.github/workflows/e2e-openshift.yaml

```diff
@@ -0,0 +1,31 @@
+name: "OpenShift end-to-end tests"
+on: [push, pull_request]
+
+jobs:
+  end-to-end:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        TEST_GROUP: [es-self-provisioned]
+    steps:
+    - uses: jpkrohling/setup-kubectl@v1-release
```

Comment:
What is this uses? Why does it reference your account?

Reply:
This downloads and stores kubectl in the local GitHub cache. It's under my personal account because it doesn't really belong to Jaeger.